api/logic/tag_janitor_test.go

```diff
@@ -0,0 +1,71 @@
+package logic
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/quintilesims/layer0/api/logic/mock_logic"
+	"github.com/quintilesims/layer0/common/db/tag_store"
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/quintilesims/layer0/common/testutils"
+)
+
+func TestTagJanitorPulse(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	taskLogicMock := mock_logic.NewMockTaskLogic(ctrl)
+	tagStore, tagsAdded := getTagStore()
```

Comment:
I think this helper function could be improved by specifying which tags to insert instead of the other way around. That would make it easier to read and understand the test, e.g.:

(taken from https://github.com/quintilesims/layer0/blob/develop/common/db/tag_store/dynamo_test.go#L14)
```
taskTags := []models.Tag{
        {EntityID: "t1", EntityType: "task", Key: "name", Value: "tsk1"},
	{EntityID: "t1", EntityType: "task", Key: "environment_id", Value: "e1"},
	{EntityID: "t2", EntityType: "task", Key: "name", Value: "tsk2"},
	{EntityID: "t2", EntityType: "task", Key: "environment_id", Value: "e2"},
}

tagStore := getTagStore(taskTags)
...

testutils.AssertEqual(t, 2, len(tags))
testutils.AssertEqual(t, "t2", tags[0].EntityID)
testutils.AssertEqual(t, "t2", tags[1].EntityID)
```